cmd/root.go

```diff
@@ -1,27 +1,29 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 	"strings"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	dbm "go.infratographer.com/tenant-api/db"
-	"go.infratographer.com/tenant-api/internal/config"
 	"go.infratographer.com/x/crdbx"
 	"go.infratographer.com/x/goosex"
 	"go.infratographer.com/x/loggingx"
 	"go.infratographer.com/x/otelx"
 	"go.infratographer.com/x/versionx"
 	"go.infratographer.com/x/viperx"
 	"go.uber.org/zap"
+
+	dbm "go.infratographer.com/tenant-api/db"
+	"go.infratographer.com/tenant-api/internal/config"
 )
 
 const appName = "tenant-api"
 
 var (
 	cfgFile string
-	logger  *zap.Logger
+	logger  *zap.SugaredLogger
```

Comment:
Most of the logs use non sugared api. Only one new log line was added using Infow, perhaps we should leave this as the normal logger and update that line?